tools/rosbag/README.md

```diff
@@ -17,6 +17,7 @@ rosbag record --file-name record.yaml
 # topic_name: freq(double)
 # topic_name: -1 (it will be recorded at publish freq)
 #           : 0 (it will not be recorded)
+#           : -2 (for latched topics, to keep latched topics in every bag split)
```

Comment:
can you update this comment? Otherwise, someone might later think that is redundant.

Reply:
Yeah, will do.